launch/ros_control.launch

```diff
@@ -0,0 +1,31 @@
+<launch>
+  <node pkg="libbarrett_ros" type="controller" name="controller" output="screen">
+
+    <rosparam command="load" subst_value="true"
+              file="$(find herb_launch)/config/ros_control.yaml"/>
```

Comment:
I don't think `subst_value` is required here anymore - we don't use any `roslaunch` variables in the YAML file.